scripts/univ3_prices.py

```diff
@@ -6,15 +6,15 @@
 import json
 
 
-POOL_NAME = 'ETH-USDC'
-
-
 def split_args(args):
     l_arg = args.split(',')
     pool_addr = l_arg[0].strip()
     lb = int(l_arg[1].strip())
     ub = int(l_arg[2].strip())
-    return pool_addr, lb, ub
+    pool_name = l_arg[3].strip()
+    twap_length = int(l_arg[4].strip())
+    periodicity = int(l_arg[5].strip())
```

Comment:
maybe could also do stripargs = lambda x: int(x[i].strip() for i in range(len(x)))

then just pool_addr, lb, ub, pool_name, twap_length, periodicity = stripargs(l_arg)